### PR TITLE
Optionally link ssh dropins

### PR DESCRIPTION
**Pull request by fbuihuu**

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -204,6 +204,7 @@ sshconfdir = get_option('sshconfdir')
 if sshconfdir == ''
         sshconfdir = sysconfdir / 'ssh/ssh_config.d'
 endif
+conf.set10('LINK_SSHPROXYDROPIN', sshconfdir != 'no' and not sshconfdir.startswith('/usr/'))
```
[the remark this answers is not in the document]
Comment (poettering) [reply meson.build:207]: ah, i see now why you need this, ignore me then.

**Comment by poettering** [comment]
lgtm

File: meson.build [hunk 2]
```diff
@@ -204,11 +204,13 @@ sshconfdir = get_option('sshconfdir')
...
 
 sshdconfdir = get_option('sshdconfdir')
 if sshdconfdir == ''
         sshdconfdir = sysconfdir / 'ssh/sshd_config.d'
 endif
+conf.set10('LINK_SSHD_USERDB_DROPIN', sshconfdir != 'no' and not sshdconfdir.startswith('/usr/'))
```
Comment (YHNdnzj) [inline meson.build]:
```suggestion
conf.set10('LINK_SSHD_USERDB_DROPIN', sshdconfdir != 'no' and not sshdconfdir.startswith('/usr/'))
```

File: meson.build [hunk 3]
```diff
@@ -204,11 +204,13 @@ sshconfdir = get_option('sshconfdir')
 if sshconfdir == ''
         sshconfdir = sysconfdir / 'ssh/ssh_config.d'
 endif
+conf.set10('LINK_SSHPROXYDROPIN', sshconfdir != 'no' and not sshconfdir.startswith('/usr/'))
```
Comment (YHNdnzj) [inline meson.build]:
```suggestion
conf.set10('LINK_SSH_PROXY_DROPIN', sshconfdir != 'no' and not sshconfdir.startswith('/usr/'))
```

File: meson.build (hunk 2 above)
Comment (fbuihuu) [reply meson.build]: good catch, thanks.

**Comment by YHNdnzj** [comment]
Hmm, this is relevant for v256 too, and it's pretty trivial. Hence I think we should merge this now?

**Comment by fbuihuu** [comment]
Agreed.

**Pull request merged**